portal-test/src/com/liferay/portal/kernel/test/util/HTTPTestUtil.java

```diff
@@ -49,6 +49,15 @@ public static JSONObject invokeToJSONObject(
 		return JSONFactoryUtil.createJSONObject(HttpUtil.URLtoString(options));
```

Comment:
I think you could reuse here your invokeToString new method to avoid duplicated code

Reply:
Make sense!!